src/routes.json

```diff
@@ -24,6 +24,18 @@
         "title": "Hour of Code"
     },
     {
+        "name": "explore",
+        "pattern": "^/explore/:projects/:all/?$",
```

Comment:
@rschamp I'm not we've yet figured out how to handle this between fastly vs express yet. Since these are now turned into varnish rules, should we be doing something like `^/explore/\d{1,15}/\w{1,30}/?$` instead (that is a rough guestimate of the rule)